newsSummaryModel.js

```diff
@@ -0,0 +1,20 @@
+class NewsSummaryModel {
+  constructor() {
+    this.headlines = [];
+  }
+
+  getNews() {
+    return this.headlines;
+  }
+
+  addNews(headline) {
+    this.reset();
+    this.headlines = headline;
```

Comment:
You don't really need a reset since you are overwriting the whole headline array here anyway.